db/migrate/20151126202932_downcase_email_and_remove_duplicate_entries.rb

```diff
@@ -0,0 +1,9 @@
+class DowncaseEmailAndRemoveDuplicateEntries < ActiveRecord::Migration
+  def change
+  	members = Member.all
+  	members.each do |member|
+  		member.email.downcase!
+  		member.destroy unless member.save
```

Comment:
Everything is good, but you should take this out. This is assuming that the reason why the `save!` fails is because of the email, so you might be destroying records that fail for some other reason. If you want to write a proper duplicate migration you need to find all records that have a duplicate e-mail, explicitly, and delete those. 

I'd merge the PR even if it doesn't deal with those dups, it's always something the admins can do by hand when the migration fails. Annoying, but such is life.